tools/devops/automation/templates/build/configure.yml

```diff
@@ -106,9 +107,11 @@ steps:
     patVariable: '$(OneLocBuild--PAT)'
 
 - task: PublishBuildArtifacts@1
-  condition: eq('$(Build.SourceBranchName)', 'merge')
+  condition: eq('$[PrBuild]', 'True')
   continueOnError: true
   inputs:
     PathtoPublish: '$(Build.ArtifactStagingDirectory)'
     ArtifactName: 'localizationDrop'
     publishLocation: 'Container'
+  env:
+    PrBuild: $(labels.prBuild)
```

Comment:
Not needed and is not doing what you think since it is a env variable for scripts.